stan/math/opencl/prim/bernoulli_lpmf.hpp

```diff
@@ -0,0 +1,115 @@
+#ifndef STAN_MATH_OPENCL_PRIM_BERNOULLI_LPMF_HPP
+#define STAN_MATH_OPENCL_PRIM_BERNOULLI_LPMF_HPP
+#ifdef STAN_OPENCL
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/log.hpp>
+#include <stan/math/prim/fun/log1m.hpp>
+#include <stan/math/prim/fun/log1p.hpp>
+#include <stan/math/opencl/kernel_generator.hpp>
+#include <stan/math/prim/functor/operands_and_partials.hpp>
+
+namespace stan {
+namespace math {
+
+/** \ingroup opencl
+ * Returns the log PMF of the Bernoulli distribution. If containers are
+ * supplied, returns the log sum of the probabilities.
+ *
+ * @tparam T_n_cl type of integer parameters
+ * @tparam T_prob_cl type of chance of success parameters
+ * @param n integer parameter
+ * @param theta chance of success parameter
+ * @return log probability or log sum of probabilities
+ * @throw std::domain_error if theta is not a valid probability
+ * @throw std::invalid_argument if container sizes mismatch.
+ */
+template <
+    bool propto, typename T_n_cl, typename T_prob_cl,
```

Comment:
propto is not in doxy. But its also not listed in CPU distributions (I checked like 5 and then stopped). So I guess this is fine.